mne/bem.py

```diff
@@ -1655,6 +1683,9 @@ def make_flash_bem(subject, overwrite=False, show=True, subjects_dir=None,
         Show surfaces to visually inspect all three BEM surfaces (recommended).
     subjects_dir : string, or None
         Path to SUBJECTS_DIR if it is not set in the environment.
+    flash_path : str | None
+        Path to the flash images. If None (default), the current folder is
+        used.
```

Comment:
indeed

it should be:

/mri/flash/parameter_maps

or maybe just :

/mri/flash/

see how it's done with sample